src/xmipp/libraries/data/mask.h

```diff
@@ -44,10 +44,10 @@ void apply_geo_cont_2D_mask(MultidimArray< double >& mask,
 /// @ingroup DataLibrary
 //@{
 
-#define INNER_MASK 1
-#define OUTSIDE_MASK 2
-#define NO_ACTIVATE 0
-#define ACTIVATE 1
+constexpr int INNER_MASK = 1;
```

Comment:
this should really be an `enum class`

Reply:
In the cases I have reviewed, always we care about the value, is compared with a int, is a int parameter for another function...